src/custom_service_account.rs

```diff
@@ -167,9 +175,14 @@ impl<'a> Claims<'a> {
         }
 
         let iat = Utc::now().timestamp();
+        let aud = if let Some(aud_str) = aud {
+            aud_str
+        } else {
+            &key.token_uri
+        };
         Claims {
             iss: &key.client_email,
-            aud: &key.token_uri,
+            aud,
```

Comment:
Would prefer to inline directly here:

```rust
aud: aud.unwrap_or(&key.token_uri),
```

Reply:
Thank you 🙏🏻 Here is the update [347cdfd](https://github.com/djc/gcp_auth/pull/119/commits/347cdfd7f2c111b1569a471a60feaa887a09c5c7)